app-frontend/src/app/components/projects/aoiDrawToolbar/index.js

```diff
@@ -144,6 +144,9 @@ class AoiDrawToolbarController {
 
```

Comment:
Changes in this file make sure that when canceling (which is now triggered by the "< back" button that all draw layers are cleared.